### PR TITLE
Final modification to home page stuff.

### PR DESCRIPTION
**Pull request by naawdawg**
Modified:
- Index.js (basically added the ajax calls and commented the code)
- Accessory.csv (added Path as column)
- SetDetails.csv (added test values for ajax requests)
- Info.php (added an additional check in getResourceAsJson)
- Memory_Model (syntax was wrong in foreach loop)

### CODE REVIEW

File: application/core/Memory_Model.php [hunk 1]
```diff
@@ -202,7 +202,7 @@ function trailing($count = 10) {
     function some($what, $which) {
 	$results = array();
 	foreach ($this->_data as $key => $record)
-	    if ($record[$what] == $which)
+	    if ($record->$what == $which)
```
Comment (aavram) [inline application/core/Memory_Model.php]: why did you have to change the core model?

Comment (naawdawg) [reply application/core/Memory_Model.php]: For some reason it kept giving me a null value of $record[$what], but worked when I used the arrow.

File: public/assets/js/index.js [hunk 2]
```diff
@@ -5,50 +5,129 @@
...
-                 optionSelected = 1;
-                 checkLoaded();
-             } else {
-                 console.log(selected);
-                 optionSelected = 2;
-                 checkLoaded();
-             }
+        $("#option").change(function (event) {
+            setId = $('#option').val();
+            if (setId == 1) {
```
Comment (aavram) [inline public/assets/js/index.js]: why can't we just pass the setId in one statement

Comment (naawdawg) [reply public/assets/js/index.js]: That works too.

Comment (naawdawg) [reply public/assets/js/index.js]: I guess just call ajaxRequest("../info/bundleDetail/" + setId); waitForAjax(); instead of doing the checks.

**Pull request merged**